File: src/main/java/com/sica/backend/dto/UserResponse.java
package com.sica.backend.dto;

import java.time.LocalDateTime;

public record UserResponse(
    String id,
    String username,
    String name,
    String firstName,
    String lastName,
    String email,
    String role,
    String foto,
    String matricula,
    String numeroEmpleado,
    Boolean activo,
    LocalDateTime creadoEn
) {}

File: src/main/kotlin/com/sica/backend/dto/UserResponse.kt
package com.sica.backend.dto

import java.time.LocalDateTime

data class UserResponse(
    val id: String,
    val username: String,
    val name: String,
    val firstName: String,
    val lastName: String,
    val email: String,
    val role: String,
    val foto: String?,
    val matricula: String? = null,
    val numeroEmpleado: String? = null,
    val activo: Boolean,
    val creadoEn: LocalDateTime
)
